source/localizable/_code-of-conduct.it.md

```diff
@@ -0,0 +1,67 @@
+Scopo
+-------
+
+Lo scopo principale di tutte le conferenze e user groups che aderiscono a questo Codice di Condotta è di essere il più inclusivi possibile rispetto al maggior numero di contributori, provenienti dai piu' eterogenei e variegati backgrounds possibili. Per questo, ci impegniamo a fornire un ambiente amichevole, sicuro e accogliente, indipendentemente dal sesso, orientamento sessuale, disabilità fisiche e/o mentali, etnia, stato socio-economico e preferenze religiose (o mancanza di tali).
```

Comment:
Changed to "partecipanti". More generic than "contributori" and we don't have the problem of the gender.